project-generated-code-backend/Backend/Model/Blog/BlogPost.cs

```diff
@@ -13,50 +13,50 @@ public class BlogPost
    {
       private String name;
       
-      private List<Comment> comment;
+      private List<Feedback> feedbacks;
```

Comment:
This was supposed to be a class for Blog that was never implemented, I am not sure if the Feedbacks are supposed to go here?